ex1/ex6/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ex6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter Maths balls:");
+            var maths = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter Phy balls:");
+            var phy = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter Chem balls:");
+            var chem = int.Parse(Console.ReadLine());
+
+            if ((maths >= 65 && phy >= 55 && chem >= 50 && chem + maths + phy >= 180) ||
+                (maths + phy >= 140 && maths + chem >= 140))
+            {
+                Console.WriteLine($"chem + maths + phy={chem + maths + phy}");
+                Console.WriteLine($"maths + phy={maths + phy}");
+                Console.WriteLine($"maths + chem={maths + chem}");
+                Console.WriteLine("You are good!");
+            }
+
+            else
+            {
+                Console.WriteLine("You are Not good!");
+                Console.WriteLine($"chem + maths + phy={chem + maths + phy}");
+                Console.WriteLine($"maths + phy={maths + phy}");
+                Console.WriteLine($"maths + chem={maths + chem}");
+            }
+            Console.ReadLine();
+
+        }
```

Comment:
- Great that you are using string interpolation for displaying data :)
- "Magic" numbers like 65, 55, 50 and 180 can be declared as const
- Try to avoid code duplication, see lines 23-25 and 32-25
- You can try to extract logic for checking the eligibility of admission for a professional course into a separate method